src/users/tests/test_api.py

```diff
@@ -3,19 +3,30 @@
 from django.urls import reverse
 
 
+@pytest.fixture
+def auth_user(django_user_model):
+    """創建一個用於認證的用戶，不影響測試邏輯"""
+    return django_user_model.objects.create(
+        email="test_auth@example.com",
+        speaker_name="Auth User",
+        verified=True,
+        is_active=True,
+    )
+
+
 @pytest.mark.django_db
-def test_user_list_with_role_filter_exact_match(api_client, django_user_model):
+def test_user_list_with_role_filter_exact_match(api_client, django_user_model, auth_user):
     group = Group.objects.create(name='Reviewer')
-    user = django_user_model.objects.create(
+    reviewer = django_user_model.objects.create(
         email="reviewer@example.com",
         speaker_name="Reviewer Name",
         bio="Some bio",
         verified=True,
         is_active=True,
     )
-    user.groups.add(group)
+    reviewer.groups.add(group)
 
-    api_client.force_authenticate(user=user)
+    api_client.force_authenticate(user=auth_user)
```

Comment:
https://github.com/pycontw/pycon.tw/pull/1247#discussion_r2178167473
why does it need force_authenticate()?
```suggestion
```